### PR TITLE
Just for discussion

### PR DESCRIPTION
**Pull request by dpshelio**
This PR includes the IDL files that are being used to correct the tilt and the orbital errors
Let's have fun @mateoi and @Cadair

### CODE REVIEW

**Comment by Cadair** [comment]
MY EYES!

File: idlcode/eis_temp_model.pro [hunk 1]
```diff
@@ -0,0 +1,228 @@
+;+
```
Comment (dpshelio) [inline idlcode/eis_temp_model.pro]: This file contains the values from the neural network.

**Comment by Cadair** [comment]
get TravisCI to test this code, I DARE YOU.

File: idlcode/eis_sts3_temp.pro [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+;       See EIS_MODEL_SERIES()
+;
+;
+; MODIFICATION HISTORY:
+;       xx-xx-2009: Suguro Kamio
+;       10-03-2010: Suguro Kamio - New version
+;       08-06-2010: Terje Fredvik - Doc header
+;-
+
+FUNCTION eis_sts3_temp, times_eis3, eis_sts3, pos_eis3, time=time
```
Comment (dpshelio) [inline idlcode/eis_sts3_temp.pro]: To extract the temperature values from the house keeping data

**Pull request closed without merging**